test/routes/graphql/Query/post.test.ts

```
@@ -0,0 +1,326 @@
+import { mercuriusClient } from "../client";
+import { Query_post, Query_signIn } from "../documentNodes";
+
+// Types
+import type {
+    ArgumentsAssociatedResourcesNotFoundExtensions,
+    InvalidArgumentsExtensions,
+    TalawaGraphQLFormattedError,
+    UnauthenticatedExtensions,
+    UnauthorizedActionOnArgumentsAssociatedResourcesExtensions,
+} from "~/src/utilities/TalawaGraphQLError";
+import { mercuriusClient } from "../client";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'mercuriusClient'. Consider to delete it or rename it.

'mercuriusClient' is defined here:


(lint/suspicious/noRedeclare)